tests/util.py

```diff
@@ -378,9 +378,13 @@ def sort_inner_lists(obj):
             return obj
 
         # drop lora-generated timestamps & users
-        del actual['fratidspunkt'], actual[
-            'tiltidspunkt'], actual[
-            'brugerref']
+        expected.pop('fratidspunkt', None)
+        expected.pop('tiltidspunkt', None)
+        expected.pop('brugerref', None)
+
+        actual.pop('fratidspunkt', None)
+        actual.pop('tiltidspunkt', None)
+        actual.pop('brugerref', None)
```

Comment:
Cosmetic, i know, but you could simplify this with:

```python
expected.update(fratidspunkt=None, tiltidspunkt=None, brugerref=None)
actual.update(fratidspunkt=None, tiltidspunkt=None, brugerref=None)
```

Doesn't `brugerref` remain the same?